lib/l10n/app_en.arb

```
@@ -1,5 +1,403 @@
+  "project_access_type_validation_error": "Choose a Project Access Type",
+  "public": "Public",
+  "private": "Private",
+  "limited_access": "Limited Access",
+  "updating_project": "Updating Project...",
+  "project_updated": "Project Updated",
+  "project_update_success": "The project was successfully updated",
+  "error": "Error",
+  "update_project": "Update Project",
+  "edit": "Edit",
+  "@edit_project_view": {},
+  "editor_picks": "संपादक की पसंद",
```

RuboCop (1.75.5)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)